scripts/examples/ex_run_commands.sh

```diff
@@ -1,15 +1,28 @@
 dbt clean
 dbt deps  || { echo "Error: dbt deps failed. Exiting..."; exit 1; }
-# dbt seed
```

Comment:
Don't merge this file. Can you add it to the root dir .gitignore? If it is already in there you may have to run something like git -rm {file} for it to ignore it now that it has been added previously. IDK for sure.